main.py

```
@@ -733,6 +778,179 @@ async def create_support_ticket(request: Request, ticket: SupportTicket):
+        }
+        
+        logger.info(f"🔐 Admin OTP request for: {data.email}")
+        response = requests.post(url, json=payload, headers=headers, timeout=15)
+        result = response.json()
+        
+        # Wix HTTP functions wrap response in 'body' property
+        if isinstance(result, dict) and "body" in result:
+            result = result["body"]
+        
+        logger.info(f"🔐 Admin OTP response: success={result.get('success')}, code={result.get('code', 'N/A')}")
+        return result
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)